crates/rome_js_analyze/src/aria_analyzers/nursery/no_redundant_alt.rs

```diff
@@ -0,0 +1,168 @@
+use rome_analyze::context::RuleContext;
+use rome_analyze::{declare_rule, Ast, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_js_syntax::jsx_ext::AnyJsxElement;
+use rome_js_syntax::{
+    AnyJsExpression, AnyJsLiteralExpression, AnyJsTemplateElement, AnyJsxAttributeValue,
+    JsSyntaxToken,
+};
+use rome_rowan::AstNode;
+
+declare_rule! {
+    /// Enforce `img` alt prop does not contain the word "image", "picture", or "photo".
+    /// The rule will first check if aria-hidden is true to determine whether to enforce the rule. If the image is
+    /// hidden, then rule will always succeed.
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <img src="src" alt="photo content" />;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <img alt={`picture doing ${things}`} {...this.props} />;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <img alt="picture of cool person" aria-hidden={false} />;
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```jsx
+    /// <img src="src" alt="alt" />;
+    /// <img src="src" alt={photo} />;
+    /// <img src="bar" aria-hidden alt="Picture of me taking a photo of an image" />;
```

Comment:
This is not valid JSX, you should wrap it into a Fragment

```suggestion
    /// <>
    /// 	<img src="src" alt="alt" />
    /// 	<img src="src" alt={photo} />
    /// 	<img src="bar" aria-hidden alt="Picture of me taking a photo of an image" />
    /// </>
```

If you accept the suggestion, you'd need to run `cargo lintdoc` again.